ingestion/src/metadata/_version.py

```diff
@@ -0,0 +1,11 @@
+"""
+Provides metadata version information.
+"""
+
+# This file is auto-generated! Do not edit!
+# Use `python -m incremental.update metadata` to change this file.
+
+from incremental import Version
+
+__version__ = Version("metadata", 0, 8, 0, dev=0)
```

Comment:
Currently, we dont have pypi package versioned `0.8.0.dev0` deployed! The checkstyle is also failing for the same.
```
Could not find a version that satisfies the requirement openmetadata-ingestion-core~=0.8.0.dev0 (from openmetadata-ingestion) (from versions: 0.6.0.dev0)
```

Reply:
This is ok. We should use `pip install -e '.'` in development and in CI, right?

Reply:
yes. That should install dependencies for incremental. However, incremental also requires `twisted` dependencies and we should add it here as well.